tests/reducers/myActivitiesReducer.test.js

```diff
@@ -0,0 +1,56 @@
+import myActivitiesReducer from '../../src/reducers/myActivitiesReducer';
+import {
+  MY_ACTIVITIES_GET_REQUEST,
+  MY_ACTIVITIES_GET_SUCCESS,
+  MY_ACTIVITIES_GET_FAILURE,
+} from '../../src/actions/constants';
+import activities from '../../src/fixtures/activities';
+
+describe('myActivitiesReducer', () => {
+  const initialState = {
+    requesting: false,
+    failed: false,
+    activities: [],
+  };
+
+  it('should return the initial state', () => {
+    expect(myActivitiesReducer(undefined, {})).toEqual({
+      requesting: false,
+      failed: false,
+      activities: [],
+    });
+  });
+
+  it('should handle MY_ACTIVITIES_GET_REQUEST', () => {
+    expect(myActivitiesReducer(initialState, {
+      type: MY_ACTIVITIES_GET_REQUEST,
+      requesting: true,
+    })).toEqual({
+      requesting: true,
+      failed: false,
+      activities: [],
+    });
+  });
+
+  it('should handle MY_ACTIVITIES_GET_REQUEST', () => {
```

Comment:
Descriptions provided do not describe exactly what is being tested, check in line 24 & 35

Reply:
Done